packages/docusaurus/README.md

```diff
@@ -0,0 +1 @@
+# Docusaurus core
```

Comment:
I didn't realize the core has _no_ README... We should definitely have something better...

Reply:
yes 😄 also surprised

this was just for the CI anyway